RSAKeyVaultProvider/HashAlgorithmExtentions.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Security.Cryptography;
+
+namespace Microsoft.Azure.KeyVault
+{
+    public static class HashAlgorithmExtentions
```

Comment:
This class should probably be internal. I don't see any utility for a consumer of the library to call this directly.

Reply:
Right, i'll fix it